rest/route/project.go

```diff
@@ -671,20 +666,19 @@ func (h *projectDeleteHandler) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.MakeJSONErrorResponder(errors.Errorf("project '%s' is already hidden", h.projectName))
 	}
 
-	if !project.UseRepoSettings {
+	if !project.UseRepoSettings() {
 		return gimlet.MakeJSONErrorResponder(
-			errors.Errorf("project '%s' must have UseRepoSettings enabled to be eligible for deletion", h.projectName))
+			errors.Errorf("project '%s' must be attached to a repo enabled to be eligible for deletion", h.projectName))
```

Comment:
should drop "enabled" from this comment

Reply:
This comment is a duplicate, right?

Reply:
😮  looks like it, not sure how that even happens

Reply:
haha me neither! I just wanted to double check!